JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/derivation/DerivationClientIntegrationTest.java

```diff
@@ -9,265 +9,258 @@
 import java.util.List;
 import java.util.Random;
 import java.util.UUID;
-import java.util.concurrent.TimeUnit;
 
 import org.json.JSONArray;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.jupiter.api.AfterAll;
-import org.testcontainers.containers.GenericContainer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
-import org.testcontainers.utility.DockerImageName;
 
-import uk.ac.cam.cares.jps.base.exception.JPSRuntimeException;
+import uk.ac.cam.cares.jps.base.BlazegraphContainer;
 import uk.ac.cam.cares.jps.base.query.RemoteStoreClient;
 
 /**
- * NOTE: all tests provided in this class are stress tests. It is recommended to run one test at a time.
- * Or you can run testCleanUpFinishedDerivationUpdateStressTest by itself first, then comment out it and
- * run the rest altogether.
+ * NOTE: all tests provided in this class are stress tests. It is recommended to
+ * run one test at a time.
+ * Or you can run testCleanUpFinishedDerivationUpdateStressTest by itself first,
+ * then comment out it and run the rest altogether.
  */
```

Comment:
Likely need to clarify this and align with disabling below

Reply:
All of these comment were pre-existing so I don't feel able to clarify them.

Reply:
Perhaps @jb2197 can comment.

Reply:
I'm going to leave this as it is for the PR as I haven't changed which tests from this class are run.